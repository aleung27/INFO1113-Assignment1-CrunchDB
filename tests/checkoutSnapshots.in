SET a 9
SNAPSHOT
SET ` 88
SNAPSHOT
CHECKOUT 1
LIST ENTRIES
SET i 8
SET a 2
CHECKOUT 2
LIST ENTRIES
CHECKOUT 1
LIST ENTRIES
