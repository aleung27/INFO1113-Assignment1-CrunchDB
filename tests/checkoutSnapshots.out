> ok

> saved as snapshot 1

> ok

> saved as snapshot 2

> ok

> a [9]

> ok

> ok

> ok

> ` [88]
a [9]

> ok

> a [9]

